orc8r/cloud/go/services/accessd/servicers/internal_servicers/accessd.go

```diff
@@ -0,0 +1,150 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+Accessd service provides the gRPC interface for cloud services & REST server to
+check & manage Identity access permissions.
+*/
+package internal_servicers
```

Comment:
package and directory names to be changed to "protected"
Import this package with an alias "protected_servicers" for readability wherever required